External/FEXCore/Source/Interface/IR/IR.json

```diff
@@ -676,6 +678,33 @@
       ]
     },
 
+    "InlineSyscall": {
+      "HasSideEffects": true,
+      "Desc": ["Dispatches a guest syscall directtly to the host syscall interface",
```

Comment:
Thanks, that's helpful :)

Small rephrase with spelling/grammar fixups:

       "Desc": ["Dispatches a guest syscall directly to the host syscall interface,",
               "bypassing the SyscallHandler class used by Syscall.",
               "This has significantly less overhead than Syscall, which needs to save JIT state first.",
               "Can only be used for syscalls that match across architecture,",
               "such as gettid (matches on x86/x86-64/Arm64).",

Not sure what the "Which can be directly called" was trying to say. Maybe you can backpatch that into my rephrase?

Reply:
Was just saying that the syscall can be directly passed through without any fixups.

Reply:
Changed the description to match yours.